public/partials/transaction_detail_small.html

```diff
@@ -1,4 +1,4 @@
-<div class="follow" layout="column" layout-fill>
+<div class="follow" style="position:relative !important;top:25px;width:100%;" layout="column" layout-fill>
```

Comment:
@nick22891 whilst we use inline styles at times, you can move this to the style.css file.

Code reviewed and change required.